web-registry/src/lib/clients/hooks/useInitClient.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+import { RegenApi } from '@regen-network/api/lib';
+
+interface Rpc {
```

Comment:
same comment as in your previous PR but this will be addressed as part of conflicts resolution

Reply:
Indeed!